src/simple_market.sol

```diff
@@ -98,16 +98,16 @@ contract SimpleMarket is EventfulMarket, DSMath {
         locked = false;
     }
 
-    function isActive(uint id) public constant returns (bool active) {
+    function isActive(uint id) public view returns (bool active) {
         return offers[id].timestamp > 0;
     }
 
-    function getOwner(uint id) public constant returns (address owner) {
+    function getOwner(uint id) public view returns (address owner) {
         return offers[id].owner;
     }
 
-    function getOffer(uint id) public constant returns (uint, ERC20, uint, ERC20) {
-      var offer = offers[id];
+    function getOffer(uint id) public view returns (uint, ERC20, uint, ERC20) {
```

Comment:
This is a more broad comment, but:

We have found it is better to store contracts and accept them as parameters as type `address`.  Then when we need to interact with them we use `ERC20(address).transfer()`. This allows a more generic interface for parameters (i.e. easier to pass in an `address` than a `ERC20` type to a function call) and easier to get back the `address`.  I checked the `keccak` and `keccak(ERC20) == keccak(address)` so I don't think this would impact external interactions or events, just might clean up the code a bit.

Reply:
This would be a nice clean up to do everywhere.
I had also did it in `after-audit-changes` branch, it will just imply even more changes in tests.
Anyway I'd suggest to do this change (if we finally want) after merging this solc 0.5 branch.

Reply:
do you think it is possible/worthwhile to resurrect the `after-audit-changes` branch or should we do this specific think on a new branch?